### PR TITLE
feat: bridge in Cmd+K

### PR DESCRIPTION
**Pull request by magiziz**
Fixes BX-1571
Figma link (if any):

## What changed (plus any additional context for devs)
- Added Bridge shortcut in Cmd+K
- Bumped the rank for Bridge and Swap so they appear before NFTs in Cmd+K

## Screen recordings / screenshots

image

## What to test

- Bridge shortcut should work in Cmd+K
- Ranking for Swap and Bridge shortcut should appear first, used this wallet for testing `0xbE957A475844c127DDD207B4ff1F63900FD13E57`

bridge_rank

### CODE REVIEW

File: static/json/languages/en_US.json [hunk 1]
```diff
@@ -495,6 +495,7 @@
       },
       "search_tags": {
         "swap": "bridge",
+        "bridge": "bridge",
```
Comment (magiziz) [inline static/json/languages/en_US.json:498]: @DanielSinclair Do you have any suggestions on what search tags we can add here for Bridge shortcut ?

Comment (DanielSinclair) [reply static/json/languages/en_US.json]: @magiziz Added a few tweaks!

**Review by estebanmino — APPROVED** [review]
lgtm!

**Review by derHowie — APPROVED** [review]
👌

**Pull request merged**